maps/utk.yml

```diff
@@ -3321,6 +3321,7 @@ properties:
       default: Rights Statement
     index_documentation: displayable, searchable, rights facet
     indexing:
+    - admin_only
```

Comment:
@markpbaggett - I believe this is the addition you slacked about today. I get that we need it to align with softserv, but I would be interested in seeing if we could remove it specifically from rights_statement in the future (I can't see a time when we'd want to restrict this info).

Reply:
@mlhale7 I'd ask Shana why this was made `admin_only` to begin with.  There is likely a reason.

Reply:
@markpbaggett - from Slack, it sounds like `admin_only` only makes restriction an option rather than a default so it shouldn't be a problem (but would be good to understand - I'll see if I can get more of a response).